Install fail

I had this issue pop up when trying to install Linter-spell

```
webworker-threads@0.7.11 install C:\Users\Jeff\AppData\Local\Temp\apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i\node_modules\linter-spell\node_modules\webworker-threads
node-gyp rebuild


C:\Users\Jeff\AppData\Local\Temp\apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i\node_modules\linter-spell\node_modules\webworker-threads>if not defined npm_config_node_gyp (node "C:\Users\Jeff\AppData\Local\atom\app-1.16.0\resources\app\apm\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Users\Jeff\AppData\Local\atom\app-1.16.0\resources\app\apm\bin\\..\node_modules\node-gyp\bin\node-gyp.js" rebuild ) 
C:\Users\Jeff\AppData\Local\Temp\apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i
`-- (empty)


(node:34484) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
(node:34484) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
(node:34484) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
(node:34484) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
(node:34484) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
(node:34484) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
(node:34484) Warning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
gyp ERR! configure error 
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
gyp ERR! stack     at failNoPython (C:\Users\Jeff\AppData\Local\atom\app-1.16.0\resources\app\apm\node_modules\node-gyp\lib\configure.js:449:14)
gyp ERR! stack     at C:\Users\Jeff\AppData\Local\atom\app-1.16.0\resources\app\apm\node_modules\node-gyp\lib\configure.js:404:11
gyp ERR! stack     at C:\Users\Jeff\AppData\Local\atom\app-1.16.0\resources\app\apm\node_modules\graceful-fs\polyfills.js:284:29
gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:123:15)
gyp ERR! System Windows_NT 10.0.15063
gyp ERR! command "C:\\Users\\Jeff\\AppData\\Local\\atom\\app-1.16.0\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\Jeff\\AppData\\Local\\atom\\app-1.16.0\\resources\\app\\apm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Jeff\AppData\Local\Temp\apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i\node_modules\linter-spell\node_modules\webworker-threads
gyp ERR! node -v v6.9.5
gyp ERR! node-gyp -v v3.4.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open 'C:\Users\Jeff\AppData\Local\Temp\apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i\package.json'
npm WARN apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i No description
npm WARN apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i No repository field.
npm WARN apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i No README data
npm WARN apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i No license field.
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Users\\Jeff\\AppData\\Local\\atom\\app-1.16.0\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\Jeff\\AppData\\Local\\atom\\app-1.16.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\Jeff\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\Jeff\\.atom\\.apmrc" "install" "C:\\Users\\Jeff\\AppData\\Local\\Temp\\d-11744-27176-k3ogw8.mze9ffajor\\package.tgz" "--runtime=electron" "--target=1.3.13" "--arch=x64" "--global-style" "--msvs_version=2015"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE

npm ERR! webworker-threads@0.7.11 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webworker-threads@0.7.11 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the webworker-threads package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs webworker-threads
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls webworker-threads
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\Jeff\AppData\Local\Temp\apm-install-dir-11744-27176-1g5h5kv.4yoy00ms4i\npm-debug.log
npm ERR! code 1
```

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "language-tags": "^1.0.5",
     "linter-spell-word-list": "^0.4.2",
     "lodash": "^4.17.4",
-    "natural": "^0.5.0",
+    "natural": "0.5.0",
     "os-locale": "^2.0.0"
   },
   "devDependencies": {

